docs/whatisbo.md

```diff
@@ -10,17 +10,17 @@ The idea that people own companies and sometimes companies own companies is well
 
 ## ...but it's people that ultimately benefit...
 
-Even when companies own companies, individuals always appear at the end of the ownership chain. They ultimately benefit from companies’ financial successes, and often share in their failures.
+Even when companies own companies, individuals almost always appear at the end of the ownership chain. They ultimately benefit from companies’ financial successes, and often share in their failures.
```

Comment:
There some instances where a BO chain won't resolve to an individual (an SOE or edge cases like the legal office of the pope). We might want to think about a glossary-like mechanism for pointing people towards "here's why we used the word 'almost' here.